oneview_redfish_toolkit/api/computer_system.py

```diff
@@ -15,10 +15,36 @@
 # under the License.
 
 import collections
+
+from oneview_redfish_toolkit.api.errors import OneViewRedfishError
 from oneview_redfish_toolkit.api.redfish_json_validator \
     import RedfishJsonValidator
 
 
+POWER_STATE_MAP = {
+    "On": {
+        "powerState": "On",
+    },
+    "ForceOff": {
+        "powerState": "Off",
+        "powerControl": "PressAndHold"
+    },
+    "GracefulShutdown": {
+        "powerState": "Off",
+        "powerControl": "MomentaryPress"
+    },
+    "GracefulRestart": {
+        "powerState": "On"
```

Comment:
We really need powerControl = Reset here; oneview did not reset when I used this command... it continued with a powered on machine.

Reply:
Fixed.